src/DesktopVirtualization/DesktopVirtualization.Autorest/README.md

```diff
@@ -47,21 +44,32 @@ In this directory, run AutoRest:
 > see https://aka.ms/autorest
 
 ``` yaml
-commit: 50175f111e9c899249e79eb082a75fb8a7aba0e2
+commit: 0feca76719343b0cb1e6a9d6064c7037827706ca
 require:
   - $(this-folder)/../../readme.azure.noprofile.md
 sanitize-names: true
 subject-prefix: 'Wvd'
 input-file:
-- $(repo)/specification/desktopvirtualization/resource-manager/Microsoft.DesktopVirtualization/stable/2023-09-05/desktopvirtualization.json
+- $(repo)/specification/desktopvirtualization/resource-manager/Microsoft.DesktopVirtualization/stable/2024-04-03/desktopvirtualization.json
 
-module-version: 2.1.0
+module-version: 6.0.0
```

Comment:
No need to update this version. It will be auto updated when running gen2main pipeline. Please revert it.